examples/julia/helloDeps.jl

```diff
@@ -0,0 +1,4 @@
+function foo(event, context)
```

Comment:
in this example you should check that installing dependencies works (using the dependencies somehow in the function)

Reply:
👌